### PR TITLE
feat: avatar

### PR DESCRIPTION
**Pull request by shashidwivedi**
Add styles for Avatar of different sizes. Avatar can have images or initials.

### CODE REVIEW

**Review by shivangisharma01 — APPROVED** [review]
The code looks fine. later on can add images to display different styles.

File: abstracts/variables.css [hunk 1]
```diff
@@ -66,6 +66,9 @@
   --space-xl: 2.5rem;
   --space-xxl: 4rem;
 
+  /* Border Radii */
+  --border-radius-full: 50%;
```
Comment (ttahm3d) [inline abstracts/variables.css]: Consider renaming this variable. May be you can use something like
```suggestion
  --border-radius-circle: 50%;
```

File: styles/foobar-ui.css [hunk 2]
```diff
@@ -66,6 +66,9 @@
   --space-xl: 2.5rem;
   --space-xxl: 4rem;
 
+  /* Border Radii */
+  --border-radius-full: 50%;
+
```
Comment (ttahm3d) [inline styles/foobar-ui.css]: This code also exists in `variables.css`. Is that a requirement?

**Review by ttahm3d — APPROVED** [review]
Looks good. A suggestion and a question. Please take a look.

File: styles/foobar-ui.css (hunk 2 above)
Comment (shashidwivedi) [reply styles/foobar-ui.css]: my foobar-ui.css is a generated stylesheet. variables.css is where the code is originally written

**Pull request merged**